src/actions/condition/Condition.ts

```diff
@@ -18,6 +18,11 @@ const CONDITIONAL_OPERATORS: Record<string, string> = {
   "^": "pow"
 };
 
+/**
+ * Sets up a conditional transformation.
+ * @memberOf Actions
```

Comment:
Is this Class a member of Actions?

Usually, the members of actions are the functions we use, such `scale()` or `fit()`